include/arch/arm/arch/32/mode/object/structures.bf

```diff
@@ -21,17 +21,28 @@ base 32
 
 -- 4k frame (these have a separate cap type as there is no room to
 -- store their size)
-block small_frame_cap {
-    field capFMappedASIDLow  10
+block small_frame_cap (
```

Comment:
Just putting this here as a reminder: we should separate out introduction of `asid_map` for other platforms from changes in the frame caps and other caps. They should be fully independent.